src/oal_agent/app/main.py

```
@@ -89,10 +89,44 @@ async def lifespan(app: FastAPI):
+    {
+        "name": "items",
+        "description": "Manage audit items and their lifecycle.",
+    },
+    {
+        "name": "users",
+        "description": "User management operations.",
+    },
+    {
+        "name": "monitoring",
+        "description": "Health checks, metrics, and application status.",
+        "externalDocs": external_docs,
```

Ruff (0.14.7)

Undefined name `external_docs`

(F821)